code/modules/cargo/packs/science.dm

```diff
@@ -108,17 +108,6 @@
 					/obj/item/clothing/gloves/color/latex/nitrile)
 	crate_name = "nitrile gloves crate"
 
-/datum/supply_pack/science/nuke_b_gone
```

Comment:
These are added in for a reason. Keep them both please

Reply:
What reason?

Reply:
Nuke kit can be used to disarm nukes.
And the shard kit to heal the SM.

Reply:
The only way you should ever be disarming that nuke is the nuclear authentification disk or you, as a traitor, pull a spicy core snatch to complete your objective. That's it.

And there is already a shard repair kit crate.